libqtile/layout/verticaltile.py

```diff
@@ -120,7 +122,16 @@ def configure(self, window, screen_rect):
             if n > 1:
                 border_width = self.border_width
             else:
-                border_width = 0
+                border_width = self.single_border_width
+
+            # margin
+            if n > 1:
+                margin = self.margin
+            else:
+                if self.single_margin != None:
+                    margin = self.single_margin
+                else:
+                    margin = self.margin
```

Comment:
```suggestion
            if n == 1 and self.single_margin is not None:
                margin = self.single_margin
            else:
                margin = self.margin
```

Reply:
That looks better, Python is great. Thanks!